flavors/standard-EXASOL-7.1.0/flavor_base/build_run/Dockerfile

```diff
@@ -26,7 +26,7 @@ COPY /exaudfclient /exaudfclient
 
 WORKDIR /exaudfclient/extension
 RUN ["/bin/bash", "-c", "source /env && bash build.sh --config no-tty --config optimize --config python --config java --config optimize-r --config fast-binary-both"]
-RUN cp -r -L bazel-bin/* /exaudf && rm -r /exaudf/external/bazel_tools/
+RUN cp -r -L bazel-bin/* /exaudf && rm -r /exaudf/external/bazel_tools/ && rm -r /exaudf/exaudfclient
```

Comment:
Only, deleting /exaudf/exaudfclient is probably not enough, you also should exclude it from the build by changing `--config fast-binary-both` to  `--config fast-binary-py3`, similar for base_test_build_run/Dockerfile